distributionviewer/core/static/js/app/components/views/chart-list.js

```diff
@@ -9,8 +9,8 @@ export default function(props) {
   }
 
   return (
-    <article className="chart-list">
+    <section className="chart-list">
```

Comment:
Oh yea I was momentarily confused with this being `#filters`. The charts are not an `<aside>` for sure.